Comp2024/src/main/java/frc/robot/lib/phoenix/CTREConfigs6.java

```diff
@@ -94,7 +94,7 @@ public static CANcoderConfiguration intakeRotaryCancoderConfig( )
     config.MagnetSensor.AbsoluteSensorRange = AbsoluteSensorRangeValue.Signed_PlusMinusHalf;
     config.MagnetSensor.SensorDirection = SensorDirectionValue.Clockwise_Positive;
     if (Robot.isReal( ))
-      config.MagnetSensor.MagnetOffset = (Robot.isComp( )) ? -0.3518 : -0.703613;
+      config.MagnetSensor.MagnetOffset = (Robot.isComp( )) ? (-0.311768 - 0.25 - 0.015) : (0.104492 - 0.25);
```

Comment:
I like having the 0.25 in the formula, but maybe make that some sore of named literal (kQuarterRotation or something that indicates that we measured from the vertical -- kVerticalOffset, etc.). Also, I don't understand where the 0.015 constant is coming from. That is also a good example of where a named literal would be self-documenting.